src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -179,6 +179,12 @@ public override bool Execute ()
 			Log.LogDebugTaskItems ("  LibraryDirectories:", LibraryDirectories);
 			Log.LogDebugTaskItems ("  LibraryJars:", LibraryJars);
 
+
+			if (!string.IsNullOrEmpty (ToolPath) && ToolPath != null) {
+				if (!File.Exists (Path.Combine (ToolPath, ToolExe)))
+					ToolPath = Path.Combine (ToolPath, "bin");
```

Comment:
...and if `$"{ToolPath}/bin/{ToolExe}"` doesn't exist, what should happen?

Perhaps `ToolTask` reports a sane error if the file doesn't exist? But I don't think so; I think e.g. on macOS it turns into a "cryptic" error 127.

I think we should do our own error checking with a decent error message.